src/get-started/quickstart.md

```diff
@@ -53,6 +53,20 @@ MinimalWordCount.java	WordCount.java
 
 For a detailed introduction to the Beam concepts used in these examples, see the [WordCount Example Walkthrough]({{ site.baseurl }}/get-started/wordcount-example). Here, we'll just focus on executing `WordCount.java`.
 
+## Runner-Specific Dependencies
```

Comment:
I don't think this should be needed. The quickstart is using examples/WordCount archetype, which already has (or, should have) this set up correctly. So, no action needed from the user.

I think this generally falls under `Ensure you've done any runner-specific setup` bullet below -- we should put it on the runner page. See #76 and #77 for good examples, IMO.